packages/core/utils/src/event-bus/__tests__/message-aggregator.spec.ts

```diff
@@ -18,6 +19,7 @@ describe("MessageAggregator", function () {
       data: { id: 999 },
     })
     aggregator.save({
+      // @ts-expect-error
```

Comment:
As per types, there is no property called `eventName` in `Event` type. For now I have suppressed the type errors. But I think we should check why there is a type mismatch.